acdh-oeaw/vicav-vue3
# Handle multiple markers at the same location or close to each other on the map

Testing steps

Currently the feature is enabled for all views by default.

1. Without grouping:
Open the profiles view
Set the zoom level to a distant one so that there are overlapping markers.
Click on a marker which is very near to another. 
You should see a popup showing nearby locations.

2. Test grouped list:
Add the Features view to the map.
Find a place which has a linguistic feature and profile as well (eg. Tunis).
Click on the marker.
You should see a popup with the data for the nearby locations grouped by Location name and data type (profile, feature etc.)

3. Default behavior
Click on a marker which doesn't have anything nearby.
The marker should opne the content right away. 

4. Zoom
Try zooming into a cluster until the markers are distant.
The marlers should not open the popup list anymore.
